MigFiles/SupportLibraries/ZWaveLib/ZWaveNode.cs

```diff
@@ -27,29 +27,85 @@
 using System.Threading;
 using System.Reflection;
 using System.Linq;
+using System.Diagnostics;
+
 using ZWaveLib.CommandClasses;
 using ZWaveLib.Values;
 
 namespace ZWaveLib
 {
 
+    public class SupportedCommandClasses : IEquatable<SupportedCommandClasses>
+    {
```

Comment:
(self-note)check this and other mods. perhaps some code can be moved out of here.